src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -2,17 +2,24 @@
     <aura:handler name="init" value="{!this}" action="{!c.init}"/>
     
     <aura:handler event="c:STG_EVT_Edit" action="{!c.toggleIsView}"/>
+    <aura:handler event="c:STG_EVT_Save" action="{!c.saveAccRecType}"/>
     
-    <aura:attribute name="namespacePrefix" type="String" />
+    <aura:attribute name="accRecTypes" type="Map" />
```

Comment:
I assume these attribs are calling getAccountRecordTypes from somewhere in the base controller for this component?